authentication/index.adoc

```diff
@@ -0,0 +1,57 @@
+[id="overview-of-authentication-authorization"]
+= Overview of authentication and authorization
+include::modules/common-attributes.adoc[]
+:context: overview-of-authentication-authorization
+
+toc::[]
+
+
+[id="authentication-overview"]
+== About authentication in {product-title}
+To control access to an {product-title} cluster, a cluster administrator can configure xref:../authentication/understanding-authentication.adoc#understanding-authentication[user authentication] and ensure only approved users access the cluster.
+
+To interact with an {product-title} cluster, users must first authenticate to the {product-title} API in some way. You can authenticate by providing an xref:../authentication/understanding-authentication.adoc#rbac-api-authentication_understanding-authentication[OAuth access token or an X.509 client certificate] in your requests to the {product-title} API.
+
+[NOTE]
+====
+If you do not present a valid access token or certificate, your request is unauthenticated and you receive an HTTP 401 error.
+====
+An administrator can configure authentication through the following tasks:
+
+* Configuring an identity provider: You can define any xref:../authentication/understanding-identity-provider.adoc#supported-identity-providers[supported identity provider in {product-title}] and add it to your cluster.
+* xref:../authentication/configuring-internal-oauth.adoc#configuring-internal-oauth[Configuring the internal OAuth server]: The {product-title} control plane includes a built-in OAuth server that determines the user’s identity from the configured identity provider and creates access tokens. You can configure the token duration and inactivity timeout, and customize the internal OAuth server URL.
+* Registering an OAuth client: {product-title} includes several xref:../authentication/configuring-oauth-clients.adoc#oauth-default-clients_configuring-oauth-clients[default OAuth clients]. You can xref:../authentication/configuring-oauth-clients.adoc#oauth-register-additional-client_configuring-oauth-clients[register and configure additional OAuth clients].
++
+[NOTE]
+====
+Users must specify either a default or custom OAuth client that receives and uses the token in their requests for the OAuth token.
+====
+
+* xref:../authentication/managing-oauth-access-tokens.adoc#managing-oauth-access-tokens[Viewing and managing user-owned OAuth access tokens].
```

Comment:
this is not necessarily controlled by the administrator but rather the user themselves. Is there a different section where this might fit?

Reply:
Comment addressed.